src/config.js

```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import debug from 'debug';
+
+config();
+
+const debugConfig = debug('config');
+
+const { PORT = '1337', HOST = 'localhost' } = process.env;
+
+//HACK: You need to tap $env:DEBUG="config" in terminal to turn on debug
+debugConfig(`PORT: ${PORT}`);
+debugConfig(`HOST: ${HOST}`);
+
+const myConfig = {
+    PORT,
+    HOST
+};
+
+export default myConfig;
+
+export {
+    PORT,
+    HOST,
+    myConfig as config,
```

Comment:
Exporting configuration as `config`here is a bit redudant here (however, it shows your deep understanding of how JS objects are exported 👍 )

I asked you to allow different **imports**, including importing the whole configuration as `config`. A tip here: you can import the default export as a named import :wink: I want to see that in tests. Feel free to send me a personal message if you stuck, don't be shy about that